ibis/common/grounds.py

```diff
@@ -211,20 +228,30 @@ def __cached_equals__(self, other):
         return result
 
 
-# TODO(kszucs): consider to move it somewhere else, like ibis/expr/core.py
-class Concrete(Immutable, Comparable, Annotable):
+class Concrete(Immutable, Comparable, Annotable, Traversable):
```

Comment:
Hm, why move `Traversable` here? I thought we decided that traversing schema/datatype fields was of low utility.

Reply:
I think it could be useful to e.g. replace certain datatypes in a schema:

```py
schema.replace({dt.bool: dt.tinyint})
```

Reply:
How much of this change requires all code that uses it to now filter on `ops.Node` everywhere?

Reply:
Going to add that feature.